environment.yml

```diff
@@ -29,6 +30,7 @@ dependencies:
   - nested-lookup
   - netcdf4
   - numpy !=1.24.3
+  - os
```

Comment:
This should be part of the Python library no? I don't think you need it

Reply:
indeed `os` is a system library :smiley:

Reply:
Removed changes